### ISSUE TITLE
Rare: "JsonParseException: Don't know how to turn {...} into component" client kicked

### ISSUE THREAD

**Issue by crashdemons** [crashdemons/DisplayItem-Spigot#18]
see https://hub.spigotmc.org/jira/browse/SPIGOT-6056 for more information

This is unrelated to GHSA-848q-2vxp-759r which is a separate issue inadvertantly found when trying to research the cause of this issue,

**Comment by crashdemons** [comment]
TL;DR - if you have this exception then you probably have plugins on your server that shade in GSON libraries without renaming their packages (their jar contains the folder path com/google/json which may be possible to delete to solve the problem, but not in all cases).

Plugins doing this were inadvertantly causing errors with unrelated plugins that format text.

examples:
NuVotifier shades this library without relocation but it has been fixed in a Dev Build
DiscordSRV previously did this but a new release has fixed the bug recently

You should update any affected plugins to resolve the issue.
